paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -739,6 +701,37 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
         )
     }
 
+    private fun USBankAccountFormScreenState.toPaymentSelection(): PaymentSelection.New.USBankAccount? {
```

Comment:
This reuses some of the behavior that was previously in `updatePaymentSelection`.